api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -77,6 +86,12 @@ def publish_cdr(cmd_name, args)
     "The Google Cloud project associated with this workbench environment, " +
     "e.g. all-of-us-rw-staging. Required."
   )
+  op.add_option(
+     "--tier [tier]",
+     ->(opts, v) { opts.tier = v},
+     "The access tier associated with this CDR, " +
+     "e.g. registered. Required."
```

Comment:
Instead of making --tier required, can we default it to registered?

Reply:
Is that desirable?  Thinking about the potential of accidentally pushing a CT CDR into the RT here.

Reply:
I don't think there is any harm in having a default tier. Regardless, this command currently won't stop you from publishing a CT CDR into the RT.

Reply:
OK, I'll add the default.  https://precisionmedicineinitiative.atlassian.net/browse/RW-6240 for protecting against publishing into the wrong tier